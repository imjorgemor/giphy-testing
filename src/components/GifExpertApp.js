import { useState } from 'react';
import AddCategory from './AddCategory';
import GifGrid from './GifGrid';


const GifExpertApp = () => {

    const [categories, setCategories] = useState("trending")

    return (
        <>
            <header>
                <h2>GifExpertApp</h2>
            </header>

            <section>
                <AddCategory setCategories={setCategories} />
            </section>

            <GifGrid
                key={categories}
                category={categories}
            />
        </>
    )
}

export default GifExpertApp