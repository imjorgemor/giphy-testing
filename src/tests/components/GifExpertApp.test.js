import GifExpertApp from "../../components/GifExpertApp";
import { shallow } from "enzyme";
import "@testing-library/jest-dom"

describe("test app", ()=>{

    const component = shallow(<GifExpertApp />)

    test("test de renderizado", ()=>{
        expect(component).toMatchSnapshot();
    })
})