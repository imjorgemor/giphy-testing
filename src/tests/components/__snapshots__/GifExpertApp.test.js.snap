// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test app test de renderizado 1`] = `
<Fragment>
  <header>
    <h2>
      GifExpertApp
    </h2>
  </header>
  <section>
    <AddCategory
      setCategories={[Function]}
    />
  </section>
  <GifGrid
    category="trending"
    key="trending"
  />
</Fragment>
`;
